bigbluebutton-html5/imports/ui/components/audio/audio-modal/component.jsx

```diff
@@ -162,22 +163,30 @@ class AudioModal extends Component {
 
   componentDidMount() {
     const {
-      joinFullAudioImmediately,
-      joinFullAudioEchoTest,
       forceListenOnlyAttendee,
       audioLocked,
+      autoJoin,
+      skipCheck,
+      skipCheckOnJoin,
+      useEchoTest,
     } = this.props;
 
-    if (joinFullAudioImmediately) {
-      this.handleJoinMicrophone();
-    }
+    if (autoJoin) {
+      if (forceListenOnlyAttendee || audioLocked) {
```

Comment:
If you are removing the check for whether listenOnly is allowed in first place, this would be a good place to re-add it